tests/test_internals/test_roi_extractors_module.py

```diff
@@ -828,33 +837,31 @@ def setUp(self):
 
         self.two_photon_series_name = "two_photon_series_name"
         self.two_photon_series_metadata = dict(
-            name=self.two_photon_series_name, imaging_plane=self.imaging_plane_name, unit="unknown"
+            name=self.two_photon_series_name, imaging_plane=self.imaging_plane_name, unit="n.a."
         )
         self.metadata["Ophys"].update(TwoPhotonSeries=[self.two_photon_series_metadata])
 
-        self.num_frames = 30
-        self.num_rows = 10
-        self.num_columns = 15
         self.imaging_extractor = generate_dummy_imaging_extractor(
             self.num_frames, num_rows=self.num_rows, num_columns=self.num_columns
         )
 
-    def test_add_two_photon_series(self):
-
-        metadata = self.metadata
-
-        add_two_photon_series(imaging=self.imaging_extractor, nwbfile=self.nwbfile, metadata=metadata)
+    def test_default_iterator(self):
```

Comment:
I think that this first test could be better called `test_default_values` as is not only the iterator that is tested. What do you think?

Reply:
yes, thank you.